inc/namespace.php

```diff
@@ -46,6 +46,24 @@ function bootstrap() {
 		add_filter( 'xmlrpc_element_limit', __NAMESPACE__ . '\\filter_xmlrpc_element_limit_handler', 999 );
 	}
 
+	if ( $config['rss'] === false ) {
+		add_action( 'do_feed', __NAMESPACE__ . '\\disable_feed', 1 );
+		add_action( 'do_feed_rdf', __NAMESPACE__ . '\\disable_feed', 1 );
+		add_action( 'do_feed_rss', __NAMESPACE__ . '\\disable_feed', 1 );
+		add_action( 'do_feed_rss2', __NAMESPACE__ . '\\disable_feed', 1 );
+		add_action( 'do_feed_atom', __NAMESPACE__ . '\\disable_feed', 1 );
+
+		// Disable comment feeds.
+		add_action( 'do_feed_rss2_comments', __NAMESPACE__ . '\\disable_feed', 1 );
+		add_action( 'do_feed_atom_comments', __NAMESPACE__ . '\\disable_feed', 1 );
+
+		// Prevent feed links from being inserted in the <head> of the page.
+		add_action( 'feed_links_show_posts_feed', '__return_false', -1 );
+		add_action( 'feed_links_show_comments_feed', '__return_false', -1 );
+		remove_action( 'wp_head', 'feed_links', 2 );
+		remove_action( 'wp_head', 'feed_links_extra', 3 );
```

Comment:
I think these may need to be hooked in later (i.e. `plugins_loaded`), not sure if default-actions.php has been loaded when this runs?

Reply:
Looks like these are hooked initially on `wp_head`  in `default-filters.php` so this seems to work fine. Why do you think it needs to be hooked later?

Reply:
I *think* this function runs before `default-filters.php` is loaded (on `altis.modules.load` or whatever the event is called), but I could be wrong on that; if it's working for you, then it's probably OK

Reply:
@rmccue what's the best way to trace this actually?

Reply:
Hmm, you are right.

Reply:
@rmccue I can't seem to figure out a work around this, do you have any suggestion or I should just leave the links be?